docs/dependency_updates.rst

```diff
@@ -196,3 +196,26 @@ diff review for ``cryptography`` 2.3 to 2.7:
 6. Comment on the PR indicating that the diff review is approved.
 
 7. Send the same content from the wiki to diff-review@python.org.
+
+
+Auditing Rust dependencies
+--------------------------
+We audit Rust crates using the Cargo Vet.  To get started:
```

Comment:
using the Cargo Vet...tool?

Reply:
Meant to drop the article, but adding the noun works too.  :-)